package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "react-capstone",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": "https://girma336.github.io/react-capstone",
```

Comment:
- [ ] when opening the application either the live version or when running it locally the user gets a blank screen, this is most likely due to this piece of code when deleting this piece of code the local version runs without any issues.